internal/internal_event_handlers.go

```diff
@@ -1005,7 +1005,9 @@ func (weh *workflowExecutionEventHandlerImpl) handleStartChildWorkflowExecutionF
 		return nil
 	}
 
-	err := fmt.Errorf("start child workflow failed: %v", attributes.GetCause())
+	err := &m.WorkflowExecutionAlreadyStartedError{
```

Comment:
Why we don't care about detail failure cause now?

Reply:
AlreadyStarted is the only cause.